targeek/guru-erp
# [HR] Implement member invitation feature

## Steps to review

- Create 2 accounts
- With 1st account, create a new org
- Invite the 2nd account to the created org
- See the 2nd in the member list with *PENDING* membership
- Sign in with the 2nd account, see in org list the pending org
- Join the pending org

Resolves #98
